client/types.ts

```diff
@@ -65,6 +65,7 @@ export interface Theme {
 interface MarketplaceThemeProductDetails {
 	product_id: number;
 	product_slug: string;
+	billing_product_slug: string;
```

Comment:
Nit: are we sure this will always be returned? Aren't there any cases where this should be `undefined`?

Reply:
Good point, I had a look at the backend code when reviewing this and it looks like what could be missing is the [`product_details` which is correctly typed here](https://github.com/Automattic/wp-calypso/blob/76755a5361c1568918acd84c24e188815148886d/client/types.ts#L41), but if the `product_details` exists, then the `billing_product_slug` will be returned according to this fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Spynff%2Qznexrgcynpr%2Qgurzr%2Qcebqhpg%2Qpngnybt.cuc%3Se%3Qqoq2378o%26sv%3Qtrg_gurzr_cebqhpg_qngn%23218-og